src/vs/base/browser/markdownRenderer.ts

```diff
@@ -301,3 +301,73 @@ function getInsaneOptions(options: { readonly isTrusted?: boolean }): InsaneOpti
 	};
 }
 
+export function RenderMarkdownAsPlaintext(markdown: IMarkdownString) {
```

Comment:
I feel this could benefit from some unit tests.

Reply:
Tests added, and plaintext renderer refined.